docs/UserGuide.md

```diff
@@ -7,10 +7,12 @@
 
 # ModCraft User Guide
 
-ModCraft is an app that provides a fast and easy way for students to track courses
+ModCraft is an app that provides a fast and easy way for NUS students to track courses
 to take to meet graduation requirements and plan courses to take. The user interacts
 with it using a CLI, and it has a GUI created with JavaFX.
```

Comment:
Add full name of CLI and GUI and have the (CLI) and (GUI), only then we can use (CLI) and (GUI) in the rest of the UG?

Reply:
Okay